frontend/src/styles/_message-list.scss

```
@@ -249,22 +249,38 @@
 }
 
 .message-table {
-  --new-mail-row-height: 45px;
+  // The height one arriving row adds: .message-row's min-height plus its
+  // bottom border. The arrival animation lifts the rows below by exactly this,
+  // so it has to follow the row box.
+  --new-mail-row-height: 49px;
   background: transparent;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)